dcgpy/_subs_weights.py

```diff
@@ -0,0 +1,37 @@
+def _subs_weights(self, expr_str):
```

Comment:
I do not understand why expr_str is needed. I was expecting this to be ex. subs_weights() then simpy would be called internally to substitute the weights symbols with the weights values and return the symbolic expression with the substituted weights